### PR TITLE
Make Recipient form extensible in the future

### PR DESCRIPTION
**Pull request by ludydoo**

### CODE REVIEW

File: pkg/api/types/casedef.go [hunk 1]
```diff
@@ -8,5 +18,5 @@ type CaseDef struct {
 	// Recipient represents the type of recipient that this case is for.
 	// For example, if the CaseDef is Colombia Individual Intake, then
 	// the Recipient could be a link to a ColombiaIndividualRecipient form.
-	Recipient FormRef `json:"recipient,omitempty"`
+	Recipient Recipient `json:"recipient,omitempty"`
```
Comment (neb42) [inline pkg/api/types/casedef.go:21]: Should this not be a `RecipientRef` rather than an actual `Recipient`?

Comment (ludydoo) [reply pkg/api/types/casedef.go:21]: Or `RecipientDef` ?

**Pull request merged**